Packages/Tracking/OpenXR/Runtime/Scripts/XRLeapProviderManager.cs

```diff
@@ -0,0 +1,58 @@
+using Leap;
+using Leap.Unity;
+using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.XR.Management;
+using UnityEngine.XR.OpenXR;
+using Ultraleap.Tracking.OpenXR;
+
+
+public class XRLeapProviderManager : PostProcessProvider
+{
+    [SerializeField] private OpenXRLeapProvider openXRLeapProvider;
+    [SerializeField] private LeapXRServiceProvider leapXRServiceProvider;
+
+    public Action<LeapProvider> OnProviderSet;
+
+    private LeapProvider _leapProvider = null;
+    public LeapProvider LeapProvider
+    {
+        get
+        {
+            return (_leapProvider == null) ? leapXRServiceProvider : _leapProvider;
+        }
+    }
+
+    [Tooltip("Forces the use of the non-OpenXR provider, using LeapC hand tracking")]
+    public bool forceLeapService;
+
+    private IEnumerator Start()
+    {
+        while (XRGeneralSettings.Instance == null) yield return new WaitForEndOfFrame();
+
+        if (XRGeneralSettings.Instance.Manager.activeLoader.name == "Open XR Loader" &&
+            OpenXRSettings.Instance.GetFeature<HandTrackingFeature>() != null &&
+            OpenXRSettings.Instance.GetFeature<HandTrackingFeature>().enabled &&
```

Comment:
I can't remember what enabled means - there was some weird discussion in OpenXR ages ago about the concept of active/enabled etc? I guess this is about the layer being enabled?

Reply:
enabled is something we are using and editing in the Feature, it's one of the blocking reasons that this PR is currently Draft as we need to confirm we can safely use '.enabled' as we hope to